packages/E2ETest/app/ImageTestPage.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {StyleSheet, View, Image, Button, requireNativeComponent} from 'react-native'
+import React, { useState } from 'react';
+import { TREE_DUMP_RESULT, SHOW_IMAGE_BORDER, IMAGE_ELEMENT } from './Consts';
+const TreeDumpControl = requireNativeComponent('TreeDumpControl');
+
+const styles = StyleSheet.create({
+  container: {
+    height:220,
+    width:450,
+  },
+  imageWithBorder: {
+    height: '100%',
+    width: '100%',
+    borderRadius: 10.0,
+    borderWidth:10,
+    borderColor: 'green',
+    backgroundColor: 'red',
+  },
+  image: {
+    height: '100%',
+    width: '100%',
+    backgroundColor: 'red',
+  },  
+  buttonContainer: {
+    backgroundColor: '#2980b6',
+    paddingVertical: 15
+  },
+  buttonText: {
+    color: '#fff',
+    textAlign: 'center',
+    fontWeight: '700'
+  },  
+  treeDumpControl: {
+    height: 150,
+    width: 450,
+    margin: 10,
+  },
+});
+
+export function ImageTestPage() {
+  const [imageWithBorder, setImageBorder] = useState(false);
+  const onOressBorder = () => {
+    var previousImageBorderState = imageWithBorder;
+    setImageBorder(!previousImageBorderState);
+ }
+  return(
+  <View style={styles.container}>
+    <Image
+      testID={IMAGE_ELEMENT}
+      style={imageWithBorder?styles.imageWithBorder:styles.image}
+      resizeMode={'center'}
+      source={{uri: 'http://facebook.github.io/react-native/img/header_logo.png'}}
```

Comment:
>http://facebook.github.io/react-native/img/header_logo.png [](start = 21, length = 58)

make it to Assert. it may introduce unnecessary delay and flakiness of testing